github/licenses.go

```diff
@@ -61,8 +61,13 @@ func (l License) String() string {
 // List popular open source licenses.
 //
 // GitHub API docs: https://developer.github.com/v3/licenses/#list-all-licenses
-func (s *LicensesService) List(ctx context.Context) ([]*License, *Response, error) {
-	req, err := s.client.NewRequest("GET", "licenses", nil)
+func (s *LicensesService) List(ctx context.Context, opt *ListOptions) ([]*License, *Response, error) {
+	u, err := addOptions("licenses", opt)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
```

Comment:
You should leave `LicensesService.List` unmodified. See https://github.com/google/go-github/issues/636#issuecomment-300674337 for rationale.